doc/source/ray-overview/installation.rst

```diff
@@ -68,6 +69,11 @@ You can install the nightly Ray wheels via the following links. These daily rele
 
   :ref:`Usage stats <ref-usage-stats>` collection is enabled by default (can be :ref:`disabled <usage-disable>`) for nightly wheels including both local clusters started via ``ray.init()`` and remote clusters via cli.
 
+.. note::
+
+  Python 3.11 support is experimental.
+
+.. _`Linux Python 3.11`: https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp311-cp311-manylinux2014_x86_64.whl
```

Comment:
What's the process for adding this artifact? The LinkCheck check if failing

Reply:
I believe we could ignore the soft failure - the wheel will get automatically uploaded to the URL if it's a CI job in master.